administrator/components/com_banners/forms/banner.xml

```diff
@@ -63,7 +63,7 @@
 
 		<field
 			name="language"
-			type="contentlanguage"
+			type="BANNERSlanguage"
```

Comment:
Really "BANNERS"language? As I look through the next few files, it seems to be a find&replace error :D

Reply:
Yes, exactly. Sorry about that :D

Reply:
No problem :) I will look and test again when you fixed this